app/views/deals/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="">
+   <div class="flex flex-row w- bg-[#3778c2] p-3 text-white items-center justify-center">
+    <div class="fixed left-3">
+    <%= link_to categories_path, class: "" do %>
+      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-chevron-left" viewBox="0 0 16 16">
+    <path fill-rule="evenodd" d="M11.354 1.646a.5.5 0 0 1 0 .708L5.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z"/>
+  </svg>
+  </div>
```

Comment:
- [ ] It would be a good thing to include a logout button/ link to give your users a better user experience to not have to navigate back back to login to other accounts.